app/assets/stylesheets/_base.scss

```diff
@@ -61,6 +61,13 @@ select {
   width: 100%;
   height: 100%;
   transform: translate(-50%, -50%);
+
+  &.-absolute {
```

Comment:
This style affects the wrapper that contains the modal and veil themselves, it's not something specific of the modal.